### PR TITLE
Meme generator #13

### PR DESCRIPTION
**Pull request by GauravGupta993**
#13 Solved. 
Made a new PR because some conflicts were appearing due to which I had to discard the commit that closed the previous PR.

### CODE REVIEW

File: src/pages/activities/RandomMemes.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    useEffect(() => {
+        generateMeme()
+    }, [])
+
+    return (
+        <div className="rmeme-root">
+            <h1>Random Meme Generator</h1>
+            <div>Enter the name of Reddit community you want to get memes from.</div>
+            <div> Some examples are animememes, dankmemes, wholesomememes, etc. Submit empty box to get random memes</div>
+            <input className="rmeme-text" placeholder="Enter name of community here" id="text" type="text" onChange={GetName}></input>
```
Comment (ken1000minus7) [inline src/pages/activities/RandomMemes.js:39]: use a more unique id than `text`, rest lgtm

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
The input doesn't seem to work. I searched by writing `chess` but it still returned random meme not related to chess in any way



Also, there is CLS (Cumulative layout shift), on hovering submit button. You can take inspiration for button from `Random Quote generator` activity.

**Comment by Ishwarendra** [comment]
> Also, there is CLS (Cumulative layout shift), on hovering submit button. You can take inspiration for button from `Random Quote generator` activity.

By CLS, I mean other `div` move due to increase in size of button. This side-effect is not good.

**Comment by GauravGupta993** [comment]
> > Also, there is CLS (Cumulative layout shift), on hovering submit button. You can take inspiration for button from `Random Quote generator` activity.
> 
> By CLS, I mean other `div` move due to increase in size of button. This side-effect is not good.

Ok, will do so. Also, API used searches for memes from posts in Reddit communities. The community name for getting chess memes is r/chessmemes so putting chessmemes in the input box should work. Also, it produces related memes after clicking generate memes after clicking submit. Just after clicking submit it produces meme related to previous topic. To solve this what I can do is not generate any meme on clicking submit and generate meme only when clicking generate meme.

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
@GauravGupta993 While you are at the cls issue, write in small font about the example.
For example, if user has to type `r/chessmemes` then there should be a text `For example: r/chessmemes` will return chess memes, so that user knows what to type in the box.

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
Whenever user submits the community name, show it somewhere so user knows which community memes he will be/is getting.

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
Also add a loader, while the data is getting fetched show some text `Loading...` or some spinner. You can see `Random Quote Generator` Activity for spinner.

**Review by Ishwarendra — APPROVED** [review]

**Pull request merged**